src/auth/computeclient.ts

```diff
@@ -68,8 +71,11 @@ export class Compute extends OAuth2Client {
     try {
       // TODO: In 2.0, we should remove the ability to configure the tokenUrl,
       // and switch this over to use the gcp-metadata package instead.
-      res = await this.transporter.request<CredentialRequest>(
-          {url, headers: {'Metadata-Flavor': 'Google'}});
+      res = await ax.request<CredentialRequest>({
+        url,
+        headers: {'Metadata-Flavor': 'Google'},
+        raxConfig: {noResponseRetries: 3, retry: 3, instance: ax}
```

Comment:
The `instance` property of `raxConfig` is not [documented](https://github.com/JustinBeckwith/retry-axios/blob/a8defe01059fff98f1b2a056fd3df62e9bc22021/README.md). What does it do?

Reply:
Yeah, so this turned out to be a funny bug.  If you use `axios.create` to create a scoped instance, you need to pass a reference to that instance into the config.  Otherwise, there's no way to figure out which instance throws the error.  I need to go update the docs and submit a PR to fix this in gcp-metadata.

Reply:
The change to the docs (and test) has been made to retry-axios:
https://github.com/JustinBeckwith/retry-axios/commit/544c2a6563c3c4df69d5d441bbaf872a0a59d83f